examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -430,48 +431,6 @@ std::string createSetupPayload()
     return result;
 };
 
-bool inRfc3986UnreservedChar(const char c)
```

Comment:
Actually QRCODE_BASE_URL is necessary, because main.cpp currently doesn't use QR Code printing from the QRCodeUtils. I know that it would be much better to use for ESP32 only QRCodeUtils, but they are making some platform specific operations when generating payload in createSetupPayload, so it cannot be simply replaced with GetQRCode from QRCodeUtils. Do we want to refactor ESP32 main file completely or leave alignment to maintainers?

Reply:
Oh, you're right. I think it's OK for now, but eventually we should probably think of unifying the code across examples.